push_docker.sh

```diff
@@ -1,42 +1,42 @@
+#! /bin/bash 
+
 DOCKER_IMAGE_FILE=$1
 REGISTRY_SERVER=$2
-REGISTRY_PORT=$3
-REGISTRY_USERNAME=$4
-REGISTRY_PASSWD=$5
-DOCKER_IMAGE_TAG=$6
+REGISTRY_USERNAME=$3
+REGISTRY_PASSWD=$4
+PLATFORM=$5
+SONIC_VERSION=$6
```

Comment:
>SONIC_VERSION [](start = 0, length = 13)

This script should be general to any docker image files, some may have no sonic version at all.

Reply:
ok

Reply:
Done.

Reply:
Suggest use environment variable or `getopts` style of optional parameter.

---
In reply to: [402515805](https://github.com/Azure/sonic-buildimage/pull/4337#discussion_r402515805) [](ancestors = 402515805)